client/components/data/query-rewind-state/index.jsx

```diff
@@ -0,0 +1,37 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import { Component } from 'react';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import { requestRewindState } from 'state/rewind/actions';
+
+export class QueryRewindState extends Component {
+	componentDidMount() {
+		this.request();
+	}
```

Comment:
Should this have an update lifecycle method for when `siteId` changes?

Reply:
updated in 0af4494e8